webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2660,7 +2664,7 @@ var userOverride = {
                 
                 // Loading zopim sdk for zendesk chat integration
                 if (kommunicate._globals.zendeskChatSdkKey) {
-                    zendeskChatService.init(kommunicate._globals.zendeskChatSdkKey);
+                    zendeskChatService.init(kommunicate._globals.zendeskChatSdkKey, preChatData);
```

Comment:
result must be available at this level. use that. no need to create a new variable `preChatData`

Reply:
its not available at this point which is why i had to declare a global variable

Reply:
can you check what all parameters are coming in this? and if we can use them
https://github.com/Kommunicate-io/Kommunicate-Web-SDK/blob/fd8e1833cf38a12db5c6e236a6b5a598c0bd2d1b/webplugin/js/app/mck-sidebox-1.0.js#L2428